EnglishFilter/sections/foreign.txt

```diff
@@ -6682,6 +6682,8 @@ zdam.xyz#%#//scriptlet("abort-current-inline-script", "fetch", "reklamy")
 otomoto.pl##a[href][data-testid="banner-link"]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/71630
 dailyweb.pl##.user-widgets > section#text-22
+! https://github.com/AdguardTeam/AdguardFilters/issues/72348
+spidersweb.pl##[id^="SpidersWeb_SW__"]
```

Comment:
Is not it hidden by `Official Polish filters for AdBlock, uBlock Origin & AdGuard`?

Reply:
I still see it with Off. Polish on Chrome
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/68852737/105977741-c8841580-60a2-11eb-994f-aac197c74845.png)</details><br/>